admin.php

```diff
@@ -25,12 +27,11 @@ function head() {
     require_once(CORE_DIR . "/page/head.php");
     $head = new Head;
     $head->info['page']['title'] = "/" . BOARD_DIR . "/ - Management Panel";
-    return $head->generate();
+    echo $head->generateAdmin();
```

Comment:
Returning is preferred in case we need to use this function in the future.

There's very little reason to echo in the function and not just echo in whatever's calling it.